src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,72 @@
+const bookmarkService = require("../services/bookmarkService");
+
+const getAllBookmarks = async (req, res) => {
+  try {
+    const bookmarks = await bookmarkService.getAllBookmarks();
+    res.status(200).json({
+      message: "Bookmarks retrieved successfully",
+      data: bookmarks,
+    });
+  } catch (error) {
+    console.log(error);
```

Comment:
What response would the client get if we simply log the error?

Reply:
Additionally, we can use the framework feature https://expressjs.com/en/guide/error-handling.html to enhance the error handling section.

Reply:
this one i didn't change in this pr as I created a different PR for error handling as we discussed previously.